src/github.ts

```diff
@@ -13,6 +13,10 @@
 // limitations under the License.
 
 import {createPullRequest} from 'code-suggester';
+import {commitAndPush} from 'code-suggester/build/src/github/commit-and-push';
+import {addLabels} from 'code-suggester/build/src/github/labels';
+import {setupLogger} from 'code-suggester/build/src/logger';
```

Comment:
I'm relying on code-suggester internal implementation, that's a bit dirty and slightly risky but I wouldn't expect this to become an issue.

We could also extract code-suggester logic if we prefer.